sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -0,0 +1,91 @@
+package io.sentry.android.fragment
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentManager.FragmentLifecycleCallbacks
+import io.sentry.Breadcrumb
+import io.sentry.IHub
+import io.sentry.SentryLevel.INFO
+
+@Suppress("TooManyFunctions")
+internal class SentryFragmentLifecycleCallbacks(private val hub: IHub) : FragmentLifecycleCallbacks() {
```

Comment:
```suggestion
internal class SentryFragmentLifecycleCallbacks(private val hub: IHub =  HubAdapter.getInstance()) : FragmentLifecycleCallbacks() {
```

so in case people wanna use the `SentryFragmentLifecycleCallbacks` directly, its possible, the `HubAdapter` is just a wrapper around the Sentry static class to make things testable

Reply:
Sure,

> in case people wanna use the SentryFragmentLifecycleCallbacks directly

so I also removed `internal` modifier. Done in 099ad04ccaa3fcfe85b20179f505995dad44293c